src/handleSubmit.js

```diff
@@ -11,9 +12,24 @@ const mergeErrors = ({ asyncErrors, syncErrors }) =>
     ? asyncErrors.merge(syncErrors).toJS()
     : { ...asyncErrors, ...syncErrors }
 
+let isImmutableList
+try {
+  // ImmutableJS isList implementation if available
+  // eslint-disable-next-line import/no-extraneous-dependencies
+  const { List } = require('immutable')
```

Comment:
This creates a new issue. Now when building an app that uses `redux-form` it includes `immutable` in the bundle #4519